### ISSUE TITLE
Undefined attribute _bw_method causes KDEMultivariate.__repr__ to crash.

### ISSUE THREAD

**Issue by fsaad** [statsmodels/statsmodels#3125]
If user does not provide `bw` to `KDEMultivariate.__init__`, when [`self._compute_bw(bw)`](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/nonparametric/kernel_density.py#L116) is invoked, the [`GenericKDE._compute_bw`](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/nonparametric/_kernel_base.py#L125-L127) method is going to see `bw` is `None` and fail to set the attribute `self._bw_method`. This causes [`KDEMultivariate.__repr__`](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/nonparametric/kernel_density.py#L126) to break because `self._bw_method` is not defined.

The function should read

```
    def _compute_bw(self, bw):
        """
        Computes the bandwidth of the data.

        Parameters
        ----------
        bw: array_like or str
            If array_like: user-specified bandwidth.
            If a string, should be one of:

                - cv_ml: cross validation maximum likelihood
                - normal_reference: normal reference rule of thumb
                - cv_ls: cross validation least squares

        Notes
        -----
        The default values for bw is 'normal_reference'.
        """

        self.bw_func = dict(
            normal_reference=self._normal_reference,
            cv_ml=self._cv_ml, cv_ls=self._cv_ls)

        if bw is None:
            bw = 'normal_reference'

        if not isinstance(bw, string_types):
            self._bw_method = "user-specified"
            res = np.asarray(bw)
        else:
            # The user specified a bandwidth selection method
            self._bw_method = bw
            bwfunc = self.bw_func[self._bw_method]
            res = bwfunc()

        return res
```

**Comment by josef-pkt** [comment]
check for 0.9. This didn't have labels and got lost

**Comment by josef-pkt** [comment]
fixed in #4504 

@fsaad Thanks for reporting and showing the correction.
Unfortunately fixing this was slow.
